wikidatarefisland/pipes/pipe_one.py

```diff
@@ -0,0 +1,47 @@
+from itertools import chain
+from wikidatarefisland.pipes import statement_filters
+from .abstract_pipe import AbstractPipe
+from wikidatarefisland.data_model import statement_filterer
+
+
+class ItemExtractorPipe(AbstractPipe):
+    def __init__(self, external_id_formatter, storage,
+                 blacklisted_properties=None):
+        self._storage = storage
+        if blacklisted_properties is None:
+            blacklisted_properties = []
+        self.blacklisted_properties = blacklisted_properties
+        self.external_id_formatter = external_id_formatter
```

Comment:
This is missing SS1 injected to it and it's mentioned to be injected here: https://github.com/wmde/reference-island#pipe-1-find-unreferenced-statements-and-urls-for-a-given-list-of-wikidata-items